README.Rmd

```diff
@@ -1,460 +1,461 @@
+where data manipulation and exploration is intuitive and consistent. On the
+other hand, **easystats** focuses more on the final stretch of the analysis:
+understanding and interpreting your results and reporting them in a manuscript
+or a report, while following best practices. You can definitely use the
+*easystats* functions in a *tidyverse* workflow!
+
+> **easystats + tidyverse =** ❤️
+
+**Can *easystats* be useful to advanced users and/or developers?**
+
+Yes, definitely! **easystats** is built in terms of modules that are general
+enough to be used inside other packages. For instance, the *insight* package is
+made to easily implement support for post-processing of pretty much all
+regression model packages under the sun. We use it in all the *easystats*
+packages, but it is also used in other non-easystats packages, such as 
+[**ggstatsplot**](https://github.com/IndrajeetPatil/ggstatsplot), [**modelsummary**](https://github.com/vincentarelbundock/modelsummary/), 
+[**ggeffects**](https://github.com/strengejacke/ggeffects), and more.
+
+**So why not in yours**? 
+
+Moreover, the *easystats* packages are very lightweight, with a minimal set of
+dependencies, which again makes it great if you want to rely on them.
+
+## Documentation
+
+### Websites
+
+Each `easystats` package has a dedicated website.
+
+For example, website for `parameters` is
+<https://easystats.github.io/parameters/>.
+
+### Blog
+
+In addition to the websites containing documentation for these packages, you can
+also read posts from `easystats` blog:
+<https://easystats.github.io/blog/posts/>.
+
+### Other learning resources
+
+In addition to these websites and blog posts, you can also check out the
+following presentations and talks to learn more about this ecosystem:
+
+<https://easystats.github.io/easystats/articles/resources.html>
+
+## Dependencies
+
+*easystats* packages are designed to be lightweight, *i.e.*, they don't have any third-party hard dependencies, other than base-R packages or other *easystats* packages!
+If you develop R packages, this means that you can safely use *easystats* packages as dependencies in your own packages, without the risk of entering the [dependency hell](https://en.wikipedia.org/wiki/Dependency_hell).
+
+```{r depnetwork}
+library(deepdep)
+
+plot_dependencies("easystats", depth = 2, show_stamp = FALSE)
+```
+
+As we can see, the only exception is the [`{see}`](https://easystats.github.io/see/) package, which is responsible for plotting and creating
+figures and relies on `{ggplot2}`, which does have a substantial number of dependencies.
+
+## Usage 
+
+```{r, include=TRUE, results="hide", echo=FALSE}
+library(zoo)
+library(cranlogs)
+library(see)
+library(datawizard)
+
+# tidverse packages
+library(lubridate)
+library(ggplot2)
+
+pkgsnms <- c("bayestestR", "correlation", "datawizard", "easystats", 
+        "effectsize", "insight", "modelbased", "performance", 
+        "parameters", "report", "see")

```

Comment:
I don't know why all of these changes appear, this is the only thing I modified

Reply:
A lot of these are whitespace changes.